src/OpenLoco/src/Ui/WindowManager.h

```diff
@@ -160,8 +160,8 @@ namespace OpenLoco::Ui::Windows
         Window* openWithFlags(uint32_t flags);
         Window* openAtTrack(const Window& main, World::TrackElement* track, const World::Pos2 pos);
         Window* openAtRoad(const Window& main, World::RoadElement* track, const World::Pos2 pos);
-        void setToTrackExtra(const Window& main, World::TrackElement* track, const uint8_t bh, const World::Pos2 pos);
-        void setToRoadExtra(const Window& main, World::RoadElement* track, const uint8_t bh, const World::Pos2 pos);
+        void setToTrackExtra(World::TrackElement* track, const uint8_t bh, const World::Pos2 pos);
+        void setToRoadExtra(World::RoadElement* track, const uint8_t bh, const World::Pos2 pos);
```

Comment:
I think these names are a little suboptimal. How about:
```suggestion
        void removeTrackExtra(World::TrackElement* track, const uint8_t bh, const World::Pos2 pos);
        void removeRoadExtra(World::RoadElement* track, const uint8_t bh, const World::Pos2 pos);
```

Reply:
I've moved them to viewport interaction. I think i put them in construction because i thought they opened the construction window but they don't. I've also renamed them.